incentivizing_rewarding_participants.adoc

```diff
@@ -1,87 +1,129 @@
+
+Not every contributor to your open source project will develop a deep level of investment in the project.
+You won't always be able to convert infrequent, casual contributors into consistent, repeat contributors—nor should that necessarily be your goal.
+More important is understanding how participants' contributions align with or reflect their motivations for _making_ those contributions—and responding accordingly.
+As https://opensource.com/article/17/10/managing-casual-contributors[researcher Ann Barcomb writes], "People who contribute to boost their CV or to fix an irritating bug are less likely to continue once their need is satisfied."
+Alternatively, however, these periodic contributors (or what Barcomb calls "episodic contributors") often seek low-complexity, easily explainable, and quickly addressable issues—the very kinds of issues long-time contributors would love to see others solve so they can focus on other matters.
+Community managers therefore shouldn't strive to create experiences _they believe_ will motivate everyone to join and contribute to the project as much as they should work to understand participants' _own_ motivations for contributing to projects, then architecting project experiences that activate and satisfy those motivations, leading (presumably) to greater engagement.
+This is critical not only because it puts community members and their needs first, but also because it allows contributors with varying degrees of time, resources, knowledge, and skill to engage a project in ways that feel comfortable for them—and feel rewarded in just the same ways others might.
+
+== Understanding a project's reward models
+
+Strategies for acknowledging and honoring various contributions to an open source project are called that project's "reward models."
+The more reward models your project constructs, the better it can engage participants with differing motivations for contributing.
+As we've already noted, everyone contributes to projects for different reasons, and the most popular projects are those that make contributors feel rewarded, valued, and respected for their contributions—whatever form they happen to take.
+Generating those feelings in contributors is a great way to build your project's reputation as a rewarding, welcoming place.
+But always remember that your community's reward models will _incentivize_ the very practices they _reward_; when people understand how a community recognizes and honors its participants, they'll engage in the behaviors they observe a community celebrate in this way.
+Community managers and architects should therefore always think carefully and act deliberately when establishing and modifying a project's particular reward model.
+What they reward—either intentionally or unintentionally—is what they'll be encouraging people to do more of.
```

Comment:
The "gaming the system" aspect of incentivization is a really important part of incentivizing successfully. 

I've seen it discussed often in software badging, where the rewards may be automated and relatively easy to obtain. If you separate badging from point systems/leaderboards, this would be a great topic of discussion.

Reply:
Here's an issue where some of us with [IEEE SA Open](https://opensource.ieee.org/community-advisory-group/community-badging/about/-/issues/26) are working on strategies to mitigate gaming.

Reply:
Thanks for this reference! I will review and attempt to reference it.